src/AutoRest.CSharp/Common/Output/Models/Serialization/Json/JsonAdditionalPropertiesSerialization.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace AutoRest.CSharp.Output.Models.Serialization.Json
 {
-    internal class JsonAdditionalPropertiesSerialization : JsonPropertySerialization
+    internal record JsonAdditionalPropertiesSerialization : JsonPropertySerialization
```

Comment:
changing this to record does not really change anything in the runtime's perspective.
But record has its own "ToString" implementation, which simplifies the debugging process.
Therefore I leave these changes in this PR.